src/cluster/ec2_pke.go

```diff
@@ -859,7 +859,8 @@ func (c *EC2ClusterPKE) GetBootstrapCommand(
 			"--kubernetes-api-server=%q "+
 			"--kubernetes-cluster-name=%q "+
 			"--kubernetes-master-mode=%q "+
-			"--kubernetes-advertise-address=0.0.0.0:6443",
+			"--kubernetes-advertise-address=0.0.0.0:6443 "+,
+			"--kubernetes-node-name=$(curl http://169.254.169.254/latest/meta-data/local-hostname) "
```

Comment:
**Required**: A comma **MUST** be added here, because otherwise it is syntactically incorrect.